freq_read/freq_read.ino

```diff
@@ -64,34 +66,43 @@ void setup() {
   attachInterrupt(digitalPinToInterrupt(pin_3.pin_num),ISR_3,CHANGE);
 
   // Initialize SD card to write to
-  Serial.print("Initializing SD card...");
+  //Serial.print("Initializing SD card...");
   if (!SD.begin(4)) {
-    Serial.println("initialization failed!");
-    while (1);
+    Serial.println("SD Card initialization failed!");
+    //while (1);
   }
-  Serial.println("initialization done.");
+  Serial.println("SD Card initialization done.");
+  SDCARD = true;
   
 }
 
 void loop() {
+  double dv = ( 5.0 * analogRead( A0 ) ) / 1024.0;
```

Comment:
A comment on what these variables are, or more descriptive variable names would be helpful👍🏻